db/migrate/20160629122413_add_database_name_to_endpoints.rb

```diff
@@ -0,0 +1,5 @@
+class AddDatabaseNameToEndpoints < ActiveRecord::Migration[5.0]
+  def change
+    add_column :endpoints, :database_name, :string
```

Comment:
can we reuse the `url` field for this? like `postgres://USER:PASSWORD@HOST:PORT/NAME`
I'm not sure if there is an official RFC for this

I think, once we open endpoint for other systems as well, they all will have individual properties. But in the end they all can be encoded into an URL, which could be read by a subclass, that lives inside the provider.

cc @Fryguy